CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Enabled filter reordering based on the `filters` attribute.
```

Comment:
The changelog will probably be read by rather non-technical users, so let's try to be as understandable as possible.
```suggestion
- Support filter reordering based on the `filters` attribute in the search shortcode.
```
(Admittedly, "indexed for searching" in #27 is not the best example for an untechnical changelog item. 😬 🙈)